tests/unit/scanners/test.javascript.js

```diff
@@ -103,6 +103,53 @@ describe('JavaScript Scanner', () => {
     expect(linterMessages.length).toEqual(0);
   });
 
+  it('should support optional chaining', async () => {
+    const code = 'const dogName = adventurer.dog?.name;';
+
+    const jsScanner = new JavaScriptScanner(code, 'code.js');
+
+    const { linterMessages } = await jsScanner.scan();
+    expect(linterMessages.length).toEqual(0);
+  });
+
+  it('should support nullish coalescing operator', async () => {
+    const code = 'const baz = 0 ?? 42;';
+
+    const jsScanner = new JavaScriptScanner(code, 'code.js');
+
+    const { linterMessages } = await jsScanner.scan();
+    expect(linterMessages.length).toEqual(0);
+  });
+
+  // eslint-disable-next-line jest/no-disabled-tests
+  it.skip('should support public class fields', async () => {
+    const code = 'class MyClass { a = 1; }';
+
+    const jsScanner = new JavaScriptScanner(code, 'code.js');
+
+    const { linterMessages } = await jsScanner.scan();
+    expect(linterMessages.length).toEqual(0);
```

Comment:
Nit, would you mind to change this to `expect(linterMessages).toEqual([])` here and in the other test case added in this PR? 
(just to make the linting error more directly visible in the CI job if the test fails, so that we may able to figure out if there is some other kind of regression that is triggering the failure by just looking to the CI job logs)